### DISCUSSION TITLE
How to remove background on paste the html string in rich-text-editor?

### DISCUSSION THREAD

**Discussion by IzatIT** [mantinedev/mantine#6627] [Q&A] [answered]
When i copy text from another site or docx styles are also copying and it's normal, but I want parse the html and remove background colors, how can i do this? 
![screen-shot2](https://github.com/user-attachments/assets/7f34f700-7a3e-4d2c-b5b7-6acc4c1f2c60)
![screen_shot1](https://github.com/user-attachments/assets/ee5cab30-1ba8-4651-8e99-73534022f56d)

**Comment by rtivital** [comment, marked as answer]
You can find examples here – https://github.com/ueberdosis/tiptap/issues/306

**Comment by IzatIT** [comment]
Thank you so much it is helped a lot

сб, 3 авг. 2024 г. в 17:21, Vitaly Rtishchev ***@***.***>:

**Comment by IzatIT** [comment]
This is my solution 
const editor  = useEditor({
      editorProps: {
            transformPastedText(text) {
                return text.replaceAll("background", " ");
            },
            transformPastedHTML(html) {
                const res = html.replaceAll("pre", "div")?.replaceAll("code", "p");
                console.log(res)
                return res
            },
        },
})
